### ISSUE TITLE
Carpets while using AAC5

### ISSUE THREAD

**Issue by Vite** [Electroid/SportPaper#63]
Since https://github.com/Electroid/SportPaper/commit/923ff1fac5ec3446edd3474b8f4c891b6226ea11 was reverted with https://github.com/Electroid/SportPaper/commit/5b5762f87c9a7dfb1f93b9ee251318cc1e75ce1d [this issue](https://github.com/Electroid/SportPaper/issues/46) is existing again.

This could be easily fixed by making https://github.com/Electroid/SportPaper/blob/master/patches/server/0191-Prevent-1.7-players-glitching-in-carpet.patch toggleable via the config.

Only AAC5 users have to disable this patch. AAC5 does not even support 1.7 so AAC5 users do not need this patch.

**Comment by linsaftw** [comment]
Or maybe you can use ViaVersion API to know if the player is using 1.7.

**Comment by Vite** [comment]
I do not understand how that should fix this issue since pure 1.8 servers are affected by it even though they do not need it.

**Comment by linsaftw** [comment]
> I do not understand how that should fix this issue since pure 1.8 servers are affected by it even though they do not need it.

Didn't it only affect 1.7 clients?

**Comment by Vite** [comment]
No, it also affects 1.8 players since the carpet height is set to 0 for all players.

**Comment by Pablete1234** [comment]
With the patch, the hitbox is handled client-side, if you set it to 0 all players can freely walk on it, 1.7 players (due to their client) will be on y=0, and 1.8+ players will (again, due to their client) walk on y=0.0625.

The complaint is some anticheats will not treat this properly
